compiler/rustc_feature/src/active.rs

```diff
@@ -393,6 +393,9 @@ declare_features! (
     (active, const_trait_impl, "1.42.0", Some(67792), None),
     /// Allows the `?` operator in const contexts.
     (active, const_try, "1.56.0", Some(74935), None),
+    /// Allows function attribute `#[coverage(on/off)]`, to control coverage
+    /// instrumentation of that function.
+    (active, coverage, "CURRENT_RUSTC_VERSION", Some(84605), None),
```

Comment:
It makes sense to name this `coverage_attribute` since it otherwise makes it seem like just the concept of code coverage is the feature, and this is already stable.

Reply:
Yes, that does make sense - I was simply mirroring the attribute's name, but I agree it might be confusing.  Will change.